proposals/csharp-8.0/default-interface-methods.md

```diff
@@ -211,10 +207,14 @@ Because interfaces may now contain executable code, it is useful to abstract com
 
 > ***Open issue***: should we permit interface methods to be `protected` or `internal` or other access? If so, what are the semantics? Are they `virtual` by default? If so, is there a way to make them non-virtual?
 
-> ***Open issue***: If we support static methods, should we support (static) operators?
+> ***Open issue***: If we support static methods, should we support (static) operators? **Decision: NO**
```

Comment:
This feels inaccurate. I think we supported operators too.

Reply:
I made the same mistake here, by not *implementing* the operator. So, the compiler interpreted it as a bad declaration for a static virtual member, which isn't supported in C# 8.